backend/social_tasks/verifiers/base.py

```
@@ -0,0 +1,112 @@
+
+
+_REGISTRY: dict[str, Verifier] = {}
+
+
+def register(verifier_cls: type[Verifier]) -> type[Verifier]:
+    """Class decorator to register a Verifier in the global registry."""
+    instance = verifier_cls()
+    if not instance.verification_type:
+        raise ValueError(
+            f'{verifier_cls.__name__} must declare verification_type'
+        )
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)